editor/gui/editor_run_bar.cpp

```diff
@@ -273,12 +273,13 @@ void EditorRunBar::play_main_scene(bool p_from_native) {
 }
 
 void EditorRunBar::play_current_scene(bool p_reload) {
+	String last_current_scene = run_current_filename; // This is necessary to have a copy of the string.
+
 	EditorNode::get_singleton()->save_default_environment();
 	stop_playing();
```

Comment:
This clears the `run_current_filename` value, see below

Reply:
The code in `stop_playing` clearing `run_current_filename` was introducted in #76026, and wasn't in the original code